getting-started/what-you-need.md

```diff
@@ -11,58 +11,50 @@ position: 3
 
 # What You Need to Use the Telerik Blazor Components
 
-This article explains the packages you need to use the Telerik UI for Blazor components, how to get them, and how to configure your project to include the Telerik Blazor components.
+This article describes the packages that provide the Telerik UI for Blazor components, how to get them, and how to manually configure your project to use them.
 
-This article is a shorter, more focused version of the Getting Started articles and targets people who are familiar with the Telerik NuGet Feed and Blazor in general. If you are looking for step-by-step tutorials, see the [Client-side Blazor - Tutorial]({%slug getting-started/client-side%}) and [Server-side Blazor - Tutorial]({%slug getting-started/server-side%}) articles.
+>tip This article is a shorter, more focused version of the [Client-Side Blazor]({%slug getting-started/client-side%}) and [Server-Side Blazor]({%slug getting-started/server-side%}) step-by-step tutorials and targets people who are familiar with the Telerik NuGet Feed and Blazor in general.
 
-If you want to add the Telerik components to an existing Blazor project, our [Convert Project Wizard for VS]({%slug getting-started-vs-integration-convert-project%}) can do this for you. If you will be starting a new project, our [New Project Wizard for VS]({%slug getting-started-vs-integration-new-project%}) or [VS Code Wizard]({%slug getting-started-vs-code-integration-overview%}) can create one for you.
+>tip Telerik UI for Blazor provides wizards that can automatically configure a project for you:
+> * You can use the [Convert Project Wizard for VS]({%slug getting-started-vs-integration-convert-project%}) to configure an existing project for the Telerik components.
+> * You can use the [New Project Wizard for VS]({%slug getting-started-vs-integration-new-project%}) or [VS Code Wizard]({%slug getting-started-vs-code-integration-overview%}) to create a pre-configured project.
 
-You can review the rest of this article for information on how things work and what is required to get the Telerik components in a project manually.
+To use the Telerik UI for Blazor, you need to:
 
+1. Get the [Telerik Blazor packages](#telerik-specific-packages) in your project.
 
->caption To use the Telerik UI for Blazor, you need to:
+1. Add the [client assets](#client-assets).
 
-1. get the [Telerik Blazor packages](#telerik-specific-packages) in your project
-
-1. add the [client assets](#client-assets)
-
-1. [set up the project](#project-configuration) to recognize the Telerik components
+1. [Set up the project](#project-configuration) to recognize the Telerik components.
 
 
 ## Telerik Specific Packages
 
-The Telerik UI for Blazor component suite requires the following Telerik-specific NuGet packages:
```

Comment:
Add links to related articles where applicable.

Reply:
There are no individual links, the next section explains how to get them, which is why I avoided the link in the first paragraph.